lib/Conversion/TTKernelToEmitC/TTKernelToEmitC.cpp

```diff
@@ -324,6 +328,20 @@ class ThreadConfigHelper {
       builder->create<emitc::IncludeOp>(loc,
                                         "compute_kernel_api/eltwise_binary.h",
                                         /*isStandard=*/false);
+      builder->create<emitc::IncludeOp>(loc,
+                                        "compute_kernel_api/tile_move_copy.h",
+                                        /*isStandard=*/false);
+      builder->create<emitc::IncludeOp>(
+          loc, "compute_kernel_api/eltwise_unary/eltwise_unary.h",
+          /*isStandard=*/false);
+      // TODO (kmitrovic) exp.h is an ExpOp-specific include. Every op has one,
+      // should be handled in general, not like this.
```

Comment:
Can you turn this into an issue? I agree, we should have a ttkernel interface method where each op needs to return required includes for itself, we can aggregate all the includes here into a set.